kubetest2/pkg/app/testers/standard/ginkgo/ginkgo.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ginkgo
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"k8s.io/test-infra/kubetest2/pkg/app/testers"
+	"k8s.io/test-infra/kubetest2/pkg/app/testers/standard/ginkgo/kubectl"
+	"k8s.io/test-infra/kubetest2/pkg/exec"
+	"k8s.io/test-infra/kubetest2/pkg/types"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	binary = "ginkgo" // TODO(RonWeber): Actually find these binaries.
+
+)
+
+var (
+	kubeRoot     string
+	platformPath string
```

Comment:
should these var become part of the `Tester` instead of global vars?

Reply:
In my mind, the Tester only knows about things that are derived from the arguments or deployer, and platformPath comes from the environment.  I admit this is a bit of an arbitrary distinction.